README.md

```diff
@@ -75,6 +75,11 @@ Finally take a glance at the API/documentation workflow below. This should be do
     3. Files showing as deleted in a `git status` are in the old repo but not in the new cookiecutter.  We took care of most these by moving over the src tree, but let's do the rest now.  Go down the list and for <filename> in the `git status` "delete" files type `cp -n ../<filepath>/<filename> ./<target_filepath>`. Note that, generally, `<filepath>` and `<target_filepath>` will be the same, but may differ in cases such as `/doc/manual/source` and `/doc/source` respectively. If there are files there we don't want, don't move them over. Whenever files are copied over using the `-n` command, you may delete them in the main repository to keep your file-tree clean.
     4. Files that have been modified exist in both places and need to be merged manually.  Do these one at a time. First open the file in pycharm, then select `Git|current file|show diff` and the differences will show up.  Select anything you want to inherit from the original file.   In many cases we don't want to bring over things that have been polished in the cookiecutter, so you are mostly looking for code specific things, such as extended descriptions of the package in README and things like that.
     5. Any files that we moved over from the old place, but put into a new location in the new repo, we need to delete them from git.  For example, files that were in `doc/manual/source/` in the old repo but are not `doc/source` we correct by typing `git add doc/manual/source`.
+12. Run pytest `python -m diffpy.<package_name>.tests` to make sure everything is working (e.g., `python -m diffpy.pdfmorph.tests`). There should be no errors if all tests passed previously when you were working on pre-commit. You may encounter deprecation warnings. There might be several possibilities:
+    1. If you see numpy deprecation warnings, you can either try to fix them (or fix them later) or install `numpy < 2` for now. Remember to add it to Github issue.
+    2. If you see pkg_resources deprecation warnings (for `__init__.py` or maybe files like `testutils.py` you moved into the `tests` folder, otherwise you might be doing something wrong), you can fix them by migrating to `importlib_resources`. Refer to the [migration guide](https://importlib-resources.readthedocs.io/en/latest/migration.html). Remember to add `importlib_resources` to `requirements/test.txt`.
```

Comment:
I only want instructions for the fix you are doing in diffpy.structure, and a note that "most pkg_resources depreation warnings will be fixed by cookiecutter, but if you are in a diffpy package using unittests and see this warning you will need to make this fix: (and give the fix)